### DISCUSSION TITLE
Setting up comitup on a configured OS

### DISCUSSION THREAD

**Discussion by ghost** [davesteele/comitup#252] [Q&A]
Hello,

I was wondering if I can use this to setup wifi on a already installed OS? I have some boards running scripts that show info on displays and have them all running. Now when I want to setup wifi on the Pi zero boards on a different location, I add a config file to the SD in the boot folder, but having the boards send an AP like this to setup would make it a lot easier!

thank you

**Comment by davesteele** [comment]
Yes, but there are many potential conflicts. Take a look at the [Installing](https://github.com/davesteele/comitup/wiki/Installing-Comitup) Wiki page.
